lms/templates/learner_dashboard/program_header_view.underscore

```diff
@@ -5,7 +5,7 @@
 </picture>
 <h2 class="hd-2 title"><%- name %></h2>
 <p class="subtitle"><%- subtitle %></p>
-<a href="" class="breadcrumb"><%- gettext('Programs') %></a>
+<a href="/dashboard/programs" class="breadcrumb"><%- gettext('Programs') %></a>
```

Comment:
@schenedx I'm going to do it separately. As I wrote above, I don't want to mix that up with this fix.